src/config.rs

```diff
@@ -155,6 +155,14 @@ impl ConfigFile {
         res
     }
 
+    /// Check whether TCP is disabled for testing.
+    pub fn tcp_disabled(&self) -> bool {
```

Comment:
Hehe, if you had squashed this commit with previous one, I wouldn't have commented several things :)